### DISCUSSION TITLE
publish branch

### DISCUSSION THREAD

**Discussion by Czegledy** [SzFMV2021-Osz/warmup#22] [General]
@pintergreg 
Az lenne a kérdésem, hogy hogyan lehet megosztani a branchet amit készítettem. Eddig ezt a hibát adja: 
![image](https://user-images.githubusercontent.com/75442587/134412942-6b00987e-4ea8-48e3-bc9c-2639d13fd691.png)
Sajnos nem vagyok egy nagy git expert

**Comment by pintergreg** [comment]
@Balazs-Czegl Pontosan az történik, amit az üzenet mond: nincs jogosultságod pusholni a repóba, mivel nem vagy tagja a szervezetnek. Amennyiben jelezted a csatlakozási szándékod, akkor ezt valahogy elkevertem és elnézést kérek érte. Ha viszont nem, akkor nem a megfelelő sorrendben haladtál végig a lépéseken. Mindenesetre most hozzáadtalak.

**Comment by Rijaspb** [comment]
Visit us on https://www.rijasexports.com/
